esmvalcore/preprocessor/_other.py

```diff
@@ -38,3 +39,93 @@ def clip(cube, minimum=None, maximum=None):
+        Dictionary with fx_variable information.
+    check_level: CheckLevels
+        Level of strictness of the checks.
+
+
+    Returns
+    -------
+    iris.cube.Cube
+        Cube with added cell measures.
+    """
+    from esmvalcore.preprocessor._io import concatenate
+    from esmvalcore.cmor.fix import fix_metadata, fix_data
+    from esmvalcore.cmor.check import cmor_check_metadata, cmor_check_data
+
+    if not fx_variables:
+        return cube
+    fx_cubes = iris.cube.CubeList()
+    for _, fx_info in fx_variables.items():
+        if not fx_info:
+            continue
+        if isinstance(fx_info['filename'], str):
+            fx_info['filename'] = [fx_info['filename']]
+        for fx_file in fx_info['filename']:
+            loaded_cube = iris.load(fx_file)
+            short_name = fx_info['short_name']
+            project = fx_info['project']
+            dataset = fx_info['dataset']
+            mip = fx_info['mip']
+            freq = fx_info['frequency']
+            loaded_cube = fix_metadata(loaded_cube, short_name=short_name,
+                                       project=project, dataset=dataset,
+                                       mip=mip, frequency=freq,
+                                       check_level=check_level)
+            fx_cubes.append(loaded_cube[0])
+
+        fx_cube = concatenate(fx_cubes)
+
+        fx_cube = cmor_check_metadata(fx_cube, cmor_table=project, mip=mip,
+                                      short_name=short_name, frequency=freq,
+                                      check_level=check_level)
+
+        fx_cube = fix_data(fx_cube, short_name=short_name, project=project,
+                           dataset=dataset, mip=mip, frequency=freq,
+                           check_level=check_level)
+
+        fx_cube = cmor_check_data(fx_cube, cmor_table=project, mip=mip,
+                                  short_name=fx_cube.var_name, frequency=freq,
+                                  check_level=check_level)
+
+        measure_name = {
+            'areacella': 'area',
+            'areacello': 'area',
+            'volcello': 'volume'
+        }
+
+        if fx_cube.var_name in measure_name.keys():
+            try:
+                fx_data = da.broadcast_to(
+                    fx_cube.core_data(), cube.shape)
+            except ValueError as exc:
+                raise ValueError(
+                    f"Frequencies of {cube.var_name} and "
+                    f"{fx_cube.var_name} cubes do not match."
+                ) from exc
+            measure = iris.coords.CellMeasure(
+                fx_data,
+                standard_name=fx_cube.standard_name,
+                units=fx_cube.units,
+                measure=measure_name[fx_cube.var_name],
+                var_name=fx_cube.var_name,
+                attributes=fx_cube.attributes)
+            cube.add_cell_measure(measure, range(0, measure.ndim))
+            logger.info('Added %s as cell measure in cube of %s.',
```

Comment:
```suggestion
            logger.debug('Added %s as cell measure in cube of %s.',
```
It's better to only use debug log messages from preprocessor functions, because the terminal will become extremely cluttered for a recipe with e.g. 50 datasets and 10 variables.